hermes-common/src/main/java/pl/allegro/tech/hermes/domain/topic/schema/CompiledSchema.java

```diff
@@ -0,0 +1,48 @@
+package pl.allegro.tech.hermes.domain.topic.schema;
+
+import java.util.Objects;
+
+public class CompiledSchema<T> {
```

Comment:
How is the schema 'compiled'? It's simply SchemaWithVersion, there is no "compilation" of schema taking place, it's just a box for these two pieces that come together. Am I missing something? Otherwise this name feels confusing.